### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by joostlek**
## What type of PR is this?

- bug

## What this PR does / why we need it:

Fix typo
## Special notes for your reviewer:

_(fill-in or delete this section)_

## Testing

_(fill-in or delete this section)_

## Release Notes

_(REQUIRED)_

```release-note
Fixes typo in rule description
```

### CODE REVIEW

**Comment by michael-genson** [comment]
Is the typo present on the en-US translation file? That's the "master" file. Everything else is managed throw Crowdin:
https://crowdin.com/project/mealie

File: frontend/lang/messages/en-GB.json [hunk 1]
```diff
@@ -296,7 +296,7 @@
     "for-type-meal-types": "for {0} meal types",
     "meal-plan-rules": "Meal Plan Rules",
     "new-rule": "New Rule",
-    "meal-plan-rules-description": "You can create rules for auto selecting recipes for you meal plans. These rules are used by the server to determine the random pool of recipes to select from when creating meal plans. Note that if rules have the same day/type constraints then the categories of the rules will be merged. In practice, it's unnecessary to create duplicate rules, but it's possible to do so.",
+    "meal-plan-rules-description": "You can create rules for auto selecting recipes for your meal plans. These rules are used by the server to determine the random pool of recipes to select from when creating meal plans. Note that if rules have the same day/type constraints then the categories of the rules will be merged. In practice, it's unnecessary to create duplicate rules, but it's possible to do so.",
```
Comment (joostlek) [inline frontend/lang/messages/en-GB.json]:
```suggestion
    "meal-plan-rules-description": "You can create rules for auto selecting recipes for you meal plans. These rules are used by the server to determine the random pool of recipes to select from when creating meal plans. Note that if rules have the same day/type constraints then the categories of the rules will be merged. In practice, it's unnecessary to create duplicate rules, but it's possible to do so.",
```

**Comment by joostlek** [comment]
A right, did not check which one was the master. But it was there as well, moved the fix to that file!

**Comment by michael-genson** [comment]
Thank you!

**Review by michael-genson — APPROVED** [review]

**Pull request merged**